typescriptSrc/test/test-interpreter.ts

```diff
@@ -115,6 +116,122 @@ describe( 'NullLiteralLabel', function() : void {
     } );
 } ) ;
 
+describe ('LambdaLabel', function() : void {
+    const paramlist = mkParameterList([mkVarDecl(mkVar("x"), mkNoTypeNd(), mkNoExpNd())]);
+    const root = mkLambda(paramlist, mkNoTypeNd(), mkExprSeq([mkNumberLiteral("1")]));
+    const vm = makeStdVMS(root);
+
+    it('should evaluate to a ClosureV', function(): void {
+        assert.check(!vm.isReady());
+        vm.advance();
+        assert.check(vm.isReady());
+        vm.advance();
+        assert.check(vm.isDone());
+        assert.check(vm.isMapped(emptyList));
+        const val = vm.getVal(emptyList);
+        assert.check(val instanceof ClosureV);
+        assert.check((val as ClosureV).getLambdaNode().label() instanceof labels.LambdaLabel);
+    });
+});
+
+describe ('CallWorldLabel - closure (no arguments)', function(): void {
+    const lambda = mkLambda(mkParameterList([]), mkNoTypeNd(), mkExprSeq([mkNumberLiteral("42")]));
+    const lambdaDecl = mkVarDecl(mkVar("f"), mkNoTypeNd(), lambda);
+    const callWorld = new PNode(new labels.CallWorldLabel("f", false), []);
+    const root = mkExprSeq([lambdaDecl, callWorld]);
+    const vm = makeStdVMS(root);
+    
+    it('should evaluate to a StringV equaling 42', function() : void {
+      while (!vm.isMapped(emptyList)){
```

Comment:
I don't think this loop will run all the way to the end, so this doesn't check that the call is mapped to the value returned.  I'd suggest running the loop until isDone is true for a second time. Or run the loop until canAdvance is false and then use getFinalValue to look at the result.